Assets/MixedRealityToolkit-Tests/MixedRealityManagerTests.cs

```diff
@@ -283,7 +283,8 @@ private static void InitializeMixedRealityManager()
 
             // Test the Manager
             Assert.IsNotNull(MixedRealityManager.Instance);
-            MixedRealityManager.Instance.ActiveProfile = ScriptableObject.CreateInstance<MixedRealityConfigurationProfile>();
+            var configuration = ScriptableObject.CreateInstance<MixedRealityConfigurationProfile>();
+            MixedRealityManager.Instance.ActiveProfile = configuration;
```

Comment:
why break this up onto different lines?

Reply:
Helped with debugging to find out what default options were set. Could also probably add test cases for the config values.